JDI.Light/JDI.Light/Utils/WebDriverUtils.cs

```diff
@@ -38,6 +38,7 @@ public static bool IsLocalVersionLatestVersion(DriverType type, string executing
                     driverBinaryName = new InternetExplorerConfig().GetBinaryName();
                     result = Path.Combine(executingPath, driverBinaryName).CheckDriverVersionFormExeAttributes(latestVersion);
                     break;
+                default: break;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this empty 'default' clause.](https://app.codacy.com/manual/42_/jdi-light-csharp/pullRequest?prid=5599701)